contracts/PositionToken.sol

```diff
@@ -5,11 +5,12 @@ import "hardhat/console.sol";
 
 import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
 import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Burnable.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 import "@openzeppelin/contracts/utils/Counters.sol";
 import "@openzeppelin/contracts/access/AccessControl.sol";
 
-contract PositionToken is ERC721, ERC721Burnable, AccessControl {
+contract PositionToken is ERC721, ERC721Burnable, ERC721Enumerable, AccessControl {
```

Comment:
This enables getting the tokenId of a given address which is useful for the tests and possibly for offchain purposes